lib/publiccloud/azure.pm

```diff
@@ -162,19 +201,56 @@ sub calc_img_version {
     return "$build.$kiwi";
 }
 
+sub generate_tags {
+    # Define tags
+    my $job_id = get_current_job_id();
+    my $openqa_url = get_required_var('OPENQA_URL');
```

Comment:
```suggestion
    my $openqa_url = get_required_var('OPENQA_URL', get_var('OPENQA_HOSTNAME'));
```

Reply:
It does not make sense. The `get_required_var()` does not have second parameter.

I have 3 options:
1) `get_var('OPENQA_URL', get_required_var('OPENQA_HOSTNAME'));`
2) `get_var('OPENQA_URL', get_var('OPENQA_HOSTNAME'));`
3) `get_required_var('OPENQA_URL');`

Can you choose one?

Reply:
> It does not make sense. The `get_required_var()` does not have second parameter.
> 
> I have 3 options:
> 
> 1. `get_var('OPENQA_URL', get_required_var('OPENQA_HOSTNAME'));`
> 2. `get_var('OPENQA_URL', get_var('OPENQA_HOSTNAME'));`
> 3. `get_required_var('OPENQA_URL');`
> 
> Can you choose one?

We use 2 in the other modules.